telegram_menu/navigation.py

```diff
@@ -256,6 +259,10 @@ def __init__(self, bot: Bot, chat: Chat, scheduler: BaseScheduler) -> None:
             replace_existing=True,
         )
 
+    @staticmethod
```

Comment:
I would prefer if ´call_update´ was a member of ´BaseMessage´ instead. 
That means reworking the abstract method as well.

Reply:
OK: just wait for an additional  commit from me: I will put `call_update` as a method of `BaseMessage` and add an additional function (inlined buttons under a message with image and caption).

Reply:
Please have a look to see if you like how it is implemented now.